### DISCUSSION TITLE
[Feature] Support Custom Order like Google Photo

### DISCUSSION THREAD

**Discussion by tzutoo** [immich-app/immich#3685] [Feature Request]
### The feature

By default, Immich sorts the photos by date, the latest ones are in the beginning and the oldest ones are in the end.

In Google Photo, you can drag the photos around (even when a photo is older than the newer one) and a custom order will be preserved.

Is it possible to record the custom order in the database of Immich?

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2208
